akka-persistence-typed/src/main/scala/akka/persistence/typed/javadsl/HandlerBuilders.scala

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.javadsl
+
+import java.util.Objects
+import java.util.function.{ BiFunction, Predicate, Supplier }
+
+import akka.annotation.InternalApi
+import akka.persistence.typed.internal.EffectImpl
+import akka.persistence.typed.javadsl.HandlerBuilders.CommandHandlerBuilderByState.CommandHandlerCase
+import akka.util.OptionVal
+
+object HandlerBuilders {
```

Comment:
I'm adding it in this object in order to create a namespace for the moment. I didn't want to remove the other CommandHandlerBuilder